README.md

```diff
@@ -20,6 +21,46 @@ V2api: [![Better Stack Badge](https://uptime.betterstack.com/status-badges/v1/mo
 <img align="center" src="https://github.com/krushnarout/digitomize/assets/129386740/3f224853-cf4b-4318-ba27-8285d1c9a0fe" alt="MSME Logo" height="auto" width="180">
 <img align="center" src="https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcROO_yLE47eeYv8eSiT3aMB1AWncngyOzGHYqptZwwckpfmH4m0" alt="MELT CD Logo" height="auto" width="180">
 
+<!--Featured In section-->
+## Featured In
+
+<table>
+
+   <tr>
+      <th>Event Logo</th>
+      <th>Event Name</th>
+      <th>Event Description</th>
+   </tr>
+   <tr>
+      <td><img src=".github/assets/gssoc24.png" width="200" height="auto" loading="lazy" alt="GSSoC 24"/></td>
+      <td>GirlScript Summer of Code</td>
+      <td>GirlScript Summer of Code is a three-month-long Open Source Program conducted every summer by GirlScript Foundation. It is an initiative to bring more beginners to Open-Source Software Development.</td>
+   </tr>
+   <tr>
+      <td><img src=".github/assets/swoc.png" width="200" height="auto" loading="lazy" alt="swoc"/></td>
+      <td>Social Winter of Code</td>
+      <td>Social Winter Of Code is the 2 month long open source program by Social India , with the aim to introduce more and more people to the world of Open source.</td>
+   </tr>
+   <tr>
+      <td><img src=".github/assets/kwoc.png" width="200" height="auto" loading="lazy" alt="kwoc"/></td>
+      <td>Kharagpur Winter of Code</td>
+      <td>Kharagpur Winter of Code is a 5-week long online program for students who are new to open source software development. The program not only helps students to get involved in open source, but also prepares them for many open source summer programs; Google Summer of Code being one of them.</td>
+   </tr>
+   <tr>
+      <td><img src=".github/assets/tublian.png" width="200" height="auto" loading="lazy" alt="tublian"/></td>
+      <td>Tublian</td>
+      <td>The AI-powered developer platform to enhance your skills by 10x, build Streetcred, contribute to open source, and become discoverable to grab opportunities.</td>
```

Comment:
Add the description about tublian internship and not what platform does, we're part of their open source internship program

Reply:
Will do